packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -547,7 +533,6 @@ export default {
                     logMethod: this.$logger.logInfo
                 });
             } catch (e) {
-                this.toggleDialogError();
```

Comment:
Have you tested that it works well when you're placing a duplicated order?

Reply:
I've tested it in storybook with the duplicate error knob and it still works. It's set here now https://github.com/justeat/fozzie-components/pull/911/files/ac174149c8afa9d3c7fa66fb089bced18c5b643e#diff-3f7bda367e6ae2f5c141559c1aec5722d6606ee4d895f7d44e9837c20f7efb3cR719